modules/registry-operator-config-resources.adoc

```diff
@@ -45,6 +45,16 @@ data:
 <1>  If the registry has the port, such as `registry-with-port.example.com:5000`.
 : should be replaced with `..`.
 
+. To configure an additional CA:
++
+----
+$ oc create configmap registry-config --from-file=<cert> -n openshift-config
+$ oc edit image.config.openshift.io cluster
+spec:
+  additionalTrustedCA:
+    name: registry-config
+----
+
```

Comment:
It is better to add how to check the change succeed:

Then you can check inside image-registry pod:
$ oc rsh image-registry-xxxxx
sh-4.2$ ls /etc/pki/ca-trust/source/anchors
<external_registry_address>  image-registry.openshift-image-registry.svc..5000  image-registry.openshift-image-registry.svc.cluster.local..5000